modules/ossm-federation-overview.adoc

```diff
@@ -0,0 +1,17 @@
+////
+This module included in the following assemblies:
+- ossm-federation.adoc
+////
+
+
+[id="ossm-federation-overview_{context}"]
+= Federation overview
+
+[role="_abstract"]
+Federation is a set of features that let you connect services between separate meshes, allowing the use of {ProductShortName} features such as authentication, authorization, and traffic management across multiple, distinct administrative domains.
+
+Implementing a federated mesh lets you run, manage, and observe a single service mesh running across multiple OpenShift clusters.  {ProductName} federation takes an opinionated approach to a multi-cluster implementation of Service Mesh that assumes _minimal_ trust between meshes.
+
+Service Mesh federation assumes that each mesh is managed individually and retains its own administrator.  The default behavior is that no communication is permitted and no information is shared between meshes. The sharing of information between meshes is on an explicit opt-in basis.  Nothing is shared in a federated mesh unless it has been configured for sharing. Support functions such as certificate generation, metrics and trace collection remain local in their respective meshes.
```

Comment:
There needs to be a serial comma in the list "such as certificate generation, metrics, and trace collection"

Reply:
Actually in this case I don't want the serial comma (trust me, I love my Oxford comma!)